src/libpmem2/vm_reservation_windows.c

```diff
@@ -13,6 +13,13 @@
 #include "pmem2_utils.h"
 #include "sys_util.h"
 
+struct pmem2_vm_reservation {
```

Comment:
you don't need this, just use vm_reservation_get_addr when needed. Same for tests.

Reply:
vm_reservation_windows.c needs the *ravl_interval* from *pmem2_vm_reservation*, I could create a getter function for it, but it would only be needed in vm_reservation_windows.c file.
There's a test case that spoils the reservation address. Likewise, I could create a setter function, which would only be needed by this test, or delete this test case entirely.

Reply:
create a getter for the windows thing and remove that test case.

Reply:
Done.